modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -0,0 +1,1494 @@
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10000`
+
+---
+
+=== cloud_storage_initial_backoff_ms
+
+Initial backoff time for exponential backoff algorithm (ms).
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `100`
+
+---
+
+=== cloud_storage_manifest_cache_size
+
+Amount of memory that can be used to handle Tiered Storage metadata.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `1048576`
+
+---
+
+=== cloud_storage_manifest_cache_ttl_ms
+
+The interval, in milliseconds, determines how long the materialized manifest can stay in the cache under contention. This setting is used for performance tuning. When the spillover manifest is materialized and stored in the cache and the cache needs to evict it, it will use 'cloud_storage_materialized_manifest_ttl_ms' value as a timeout. The cursor that uses the spillover manifest uses this value as a TTL interval after which it stops referencing the manifest making it available for eviction. This only affects spillover manifests under contention.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10000`
+
+---
+
+=== cloud_storage_manifest_max_upload_interval_sec
+
+Minimum interval, in seconds, between partition manifest uploads. Actual time between uploads may be greater than this interval. If this is `null`, metadata will be updated after each segment upload.
+
+*Requires restart:* No
+
+*Optional:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17179869184`, `17179869183`]
+
+*Default:* `60`
+
+---
+
+=== cloud_storage_manifest_upload_timeout_ms
+
+Manifest upload timeout (ms).
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10000`
+
+---
+
+=== cloud_storage_max_concurrent_hydrations_per_shard
+
+Maximum concurrent segment hydrations of remote data per CPU core.  If unset, value of `cloud_storage_max_connections / 2` is used, which means that half of available S3 bandwidth could be used to download data from S3. If the cloud storage cache is empty every new segment reader will require a download. This will lead to 1:1 mapping between number of partitions scanned by the fetch request and number of parallel downloads. If this value is too large the downloads can affect other workloads. In case of any problem caused by the tiered-storage reads this value can be lowered. This will only affect segment hydrations (downloads) but won't affect cached segments. If fetch request is reading from the tiered-storage cache its concurrency will only be limited by available memory.
```

Comment:
```suggestion
Maximum concurrent segment hydrations of remote data per CPU core.  If unset, the value of `cloud_storage_max_connections / 2` is used, which means that half of available S3 bandwidth could be used to download data from S3. If the object storage cache is empty, every new segment reader requires a download. This leads to 1:1 mapping between the number of partitions scanned by the fetch request and the number of parallel downloads. If this value is too large, the downloads can affect other workloads. In case of any problem caused by the tiered-storage reads, this value can be lowered. This only affects segment hydrations (downloads), not cached segments. If a fetch request is reading from the Tiered Storage cache, its concurrency is only limited by available memory.
```

Reply:
@Deflaimun You should find about 5 remaining instances of "S3" in this file. I think they all apply to any object storage provider. In other words, here it would say "If unset, the value of `cloud_storage_max_connections /2` is used, which means that half of available object storage bandwidth could be used to download data from object storage. But, I am not sure. If each instance does apply only to S3, then please change it to "AWS". If it applies to any provider, please change it to "object storage".

Reply:
never mind: I'm pretty sure they're all generic object storage, so I've suggested changes for all of them. For this property description, I also suggest changing "S3" to "object storage".